tests/test_core.py

```diff
@@ -32,20 +32,23 @@ def test_sort():
     cleanup()
     try:
         with gpsdio.open("in.msg", "w") as f:
-            for i in xrange(0, 1000):
+            for i in range(0, 1000):
```

Comment:
Why the change (without using six)? This means a 1000 item array is opened in python 2...